src/OpenToolkit.Mathematics/Geometry/Box3.cs

```diff
@@ -0,0 +1,234 @@
+//
+// Box3.cs
+//
+// Copyright (C) 2019 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+using System;
+using System.Globalization;
+using System.Runtime.InteropServices;
+
+namespace OpenToolkit.Mathematics
+{
+    /// <summary>
+    /// Defines a 3d box.
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    public struct Box3 : IEquatable<Box3>
+    {
+        /// <summary>
+        /// The left boundary of the structure.
+        /// </summary>
+        public float Left;
```

Comment:
I don't know if you just didn't get to it yet. But I did mean using Min/Max vectors for these as well ;)

Reply:
By now i have only done 1 of the box2's wanted to get the first one down before i started working on the rest of them.